components/fxa-client/ios/FxAClient/Errors/FxAError.swift

```diff
@@ -8,22 +8,22 @@ import Foundation
 // swiftlint:disable identifier_name
```

Comment:
You need to delete these two lines for the "Check Swift formatting" to succeed.

Reply:
Done. Test passed. Thanks!